src/next-collection-view.js

```diff
@@ -112,7 +112,14 @@ const CollectionView = Backbone.View.extend({
   },
 
   _onCollectionReset() {
-    this.render();
+    this._destroyChildren();
```

Comment:
Previously reset just called render.. it was convienent, but not _really_ what needed to happen.  This duplicates very little of the render, but _only_ does what needs to occur and doesn't mess with the CompositeView.  So this _is_ a breaking change as the `render` events will not be triggered... but the regular events related to children will be.  This will also make the reset consistent between collection and composite views which currently v3 it is not.